File: lib/domain/entities/project.dart
class Project {
  const Project({
    required this.id,
    required this.title,
    required this.description,
    required this.projectUrl,
    required this.imageUrl,
    required this.projectCategory,
  });

  final int id;
  final String title;
  final String description;
  final String projectUrl;
  final String imageUrl;
  final String projectCategory;
}

File: backend/domain/entities/project.js
class Project {

    constructor(id, title, description, imageUrl, projectUrl, projectCategory) {
        this.id = id;
        this.title = title;
        this.description = description;
        this.imageUrl = imageUrl;
        this.projectUrl = projectUrl;
        this.projectCategory = projectCategory;
    };

}

module.exports = Project;